test/tcp/TcpServer.cpp

```diff
@@ -72,16 +72,16 @@ class ServerRequestHandler : public DefaultRequestHandler {
 
   void handleFireAndForgetRequest(Payload request) override {
     LOG(INFO) << "ServerRequestHandler.handleFireAndForgetRequest " << request
-              << "\n";
+              << std::endl;
```

Comment:
That is gone already, just showing because of github reviews AFAIK.  I will merge this is it passes tests.